### PR TITLE
Minimal Kafka as single yaml

### PR DESCRIPTION
**Pull request by solsson**
Meant to replace #44 and #118, to be used for testing etc.

### CODE REVIEW

**Comment by syncharny** [comment]
Looks like it may have some issues...minikafka keeps restarting:

> [2018-12-19 23:34:08,193] INFO [ZooKeeperClient] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2018-12-19 23:34:08,201] INFO Opening socket connection to server minizoo/10.1.1.8:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2018-12-19 23:34:14,196] INFO [ZooKeeperClient] Closing. (kafka.zookeeper.ZooKeeperClient)
[2018-12-19 23:34:14,291] WARN Client session timed out, have not heard from server in 6095ms for sessionid 0x0 (org.apache.zookeeper.ClientCnxn)
[2018-12-19 23:34:14,393] INFO Session: 0x0 closed (org.apache.zookeeper.ZooKeeper)
[2018-12-19 23:34:14,394] INFO EventThread shut down for session: 0x0 (org.apache.zookeeper.ClientCnxn)
[2018-12-19 23:34:14,396] INFO [ZooKeeperClient] Closed. (kafka.zookeeper.ZooKeeperClient)
[2018-12-19 23:34:14,401] ERROR Fatal error during KafkaServer startup. Prepare to shutdown (kafka.server.KafkaServer)
kafka.zookeeper.ZooKeeperClientTimeoutException: Timed out waiting for connection while in state: CONNECTING
	at kafka.zookeeper.ZooKeeperClient.$anonfun$waitUntilConnected$3(ZooKeeperClient.scala:268)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
	at kafka.utils.CoreUtils$.inLock(CoreUtils.scala:251)
	at kafka.zookeeper.ZooKeeperClient.waitUntilConnected(ZooKeeperClient.scala:264)
	at kafka.zookeeper.ZooKeeperClient.&lt;init&gt;(ZooKeeperClient.scala:97)
	at kafka.zk.KafkaZkClient$.apply(KafkaZkClient.scala:1694)
	at kafka.server.KafkaServer.createZkClient$1(KafkaServer.scala:348)
	at kafka.server.KafkaServer.initZkClient(KafkaServer.scala:372

**Comment by solsson** [comment]
It's been working multiple times for me. Did you look at the zookeeper logs? Could it be #230 ?

File: minikafka/minikafka.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: minizoo
+spec:
+  selector:
+    app: minizoo
+  ports:
+  - port: 9092
```
Comment (drysds) [inline minikafka/minikafka.yaml]: in my case, i need to change this port to 2181

**Comment by solsson** [comment]
Deprecated in favor of https://github.com/Yolean/kubernetes-kafka/pull/253

**Pull request closed without merging**